--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -266,7 +266,7 @@
         get_target_property(CODEC2_INCLUDE_DIRS codec2 INTERFACE_INCLUDE_DIRECTORIES)
         message(STATUS "  codec2 library: ${CODEC2_LIBRARY}")
         message(STATUS "  codec2 headers: ${CODEC2_INCLUDE_DIRS}")
-        find_package(lpcnetfreedv REQUIRED)
+        find_package(lpcnetfreedv)
     else()
         # Try to find manually
         find_path(CODEC2_INCLUDE_DIRS codec2.h
--- a/src/CMakeLists.txt
+++ b/src/CMakeLists.txt
@@ -53,7 +53,7 @@
 add_executable(freedv WIN32 ${FREEDV_SOURCES} ${RES_FILES})
 
 # Link imported or build tree targets.
-target_link_libraries(freedv codec2 lpcnetfreedv)
+target_link_libraries(freedv codec2)
 
 # Add build dependencies for interally built external libraries.
 if(USE_INTERNAL_CODEC2)
